libs/common-services/src/main/java/uk/gov/di/ipv/core/library/persistence/item/IpvSessionItem.java

```
@@ -42,7 +41,6 @@ public class IpvSessionItem implements PersistenceItem {
     private long ttl;
     private String emailAddress;
     private ReverificationStatus reverificationStatus;
-    private AisInterventionType aisInterventionType;
```

Is it safe to just delete this? Will there be any issues with journeys that started before this code goes live but finish after it's live?

---

Yes, shouldn't interrupt any in-flight journeys as it will be checked against `reproveIdentity` flag and new fetched account intervention type where the flag is always presented in `ClientOAuthSessionItem`.

---

I was more worried about desrialising the object from the data base and it having extra data that we no longer have a field for.
Googling hasn't found a definitive bit of documentation, but it looks like it should be safe and the extra data will be silently ignored.

---

I tried to look for an answer but couldn't find either. Previously I deleted `AccountInterventionState` and nothing breaks so it should behave in this case exactly same.